Compound_Risk_database.R

```diff
@@ -1421,191 +1428,122 @@ reign_start <- reign_data %>%
   left_join(., acled, by = "Country") %>%
-  select(Countryname, Final_score, pol_trigger_norm, z_idps_norm, fatal_z_norm) 
+  dplyr::select(Countryname, FCV_normalised, pol_trigger_norm, z_idps_norm, fatal_z_norm) 
 
 conflict_dataset <- conflict_dataset_raw %>%
   mutate(
-    flag_count = as.numeric(unlist(row_count(.,
-                                             Final_score:fatal_z_norm,
-                                             count = 10,
-                                             append = F
+    flag_count = as.numeric(unlist(row_count(
+      .,
+      pol_trigger_norm:fatal_z_norm,
+      count = 10,
+      append = F
     ))),
     fragile_1_flag = case_when(
       flag_count >= 1 ~ 10,
-      TRUE ~ apply(conflict_dataset_raw %>% select(Final_score:fatal_z_norm), 1, FUN = max, na.rm = T)
-    ),
-    fragile_2_flags = case_when(
-      flag_count >= 2 ~ 10,
-      TRUE ~ 0
-    ),
-    fragile_flag_seq = case_when(
-      flag_count >= 2 ~ 10,
-      flag_count == 1 ~ 7,
-      TRUE ~ 0
-    )
-  ) %>%
-  rename(FCS_FSI_Normalised = Final_score, REIGN_Normalised = pol_trigger_norm,
+      TRUE ~ suppressWarnings(apply(conflict_dataset_raw %>% dplyr::select(pol_trigger_norm:fatal_z_norm), 
+                   1,
+                   FUN = max,
+                   na.rm = T)
+    )),
+    fragile_1_flag = case_when(
+      fragile_1_flag == -Inf ~ NA_real_,
+      TRUE ~ fragile_1_flag
+    )) %>%
+  rename(FCS_Normalised = FCV_normalised, REIGN_Normalised = pol_trigger_norm,
          Displaced_UNHCR_Normalised = z_idps_norm, BRD_Normalised = fatal_z_norm,
-         Number_of_High_Risk_Flags = flag_count, Overall_Conflict_Risk_Score = fragile_1_flag) %>%
-  select(-fragile_2_flags, -fragile_flag_seq)
+         Number_of_High_Risk_Flags = flag_count, Overall_Conflict_Risk_Score = fragile_1_flag) 
 
-#-------------------------------------FRAGILITY SHEET--------------------------------------
+#-------------------------------------—Create Fragility sheet--------------------------------------
 # Compile joint database
 countrylist <- read.csv("https://raw.githubusercontent.com/ljonestz/compoundriskdata/master/Indicator_dataset/countrylist.csv")
 
-fragilitysheet <- left_join(countrylist, conflict_dataset, by = "Countryname") %>%
-  select(-X, -Number_of_High_Risk_Flags) %>%
+fragility_sheet <- left_join(countrylist, conflict_dataset, by = "Countryname") %>%
+  dplyr::select(-X, -Number_of_High_Risk_Flags) %>%
   rename_with(
     .fn = ~ paste0("Fr_", .), 
     .cols = colnames(.)[!colnames(.) %in% c("Country", "Countryname") ]
   )
 
-write.csv(fragilitysheet, "Risk_sheets/fragilitysheet.csv")
+write.csv(fragility_sheet, "Risk_sheets/fragilitysheet.csv")
+print("Fragility sheet written")
 
 #
 ##
 ### ********************************************************************************************
-####    CREATE ACAPS SHEET USING A RANGE OF SOURCE INDICATORS
+####    WRITE MINIMAL DIMENSION SHEETS ----
```

Comment:
New section that writes a .csv for each dimension that only includes the variables used by the monitor